packages/communication-react/review/communication-react.api.md

```diff
@@ -205,6 +205,16 @@ export type AzureCommunicationChatAdapterArgs = {
     threadId: string;
 };
 
+// @public (undocumented)
+export interface BaseCompositeProps {
+    fluentTheme?: PartialTheme | Theme;
+    icons?: CompositeIcons;
```

Comment:
Composite icons will vary per composite - e.g. call composite won't have messageSend icon - I think we should explicitly state icons per composite and not have them inherit all default icons

Reply:
Maybe using the `Pick` syntax to select the icons out of the default icons object for use in the composite to ensure no non-existent icons

Reply:
cc @anjulgarg